stub-runner/stub-runner/src/main/groovy/com/ofg/stub/StubRunner.groovy

```diff
@@ -1,70 +1,31 @@
 package com.ofg.stub
+
 import com.google.common.base.Optional
 import com.ofg.stub.mapping.ProjectMetadata
 import com.ofg.stub.mapping.StubRepository
 import com.ofg.stub.registry.StubRegistry
 import com.ofg.stub.server.AvailablePortScanner
 import com.ofg.stub.server.ZookeeperServer
+import groovy.transform.CompileStatic
 import groovy.util.logging.Slf4j
-import org.kohsuke.args4j.CmdLineException
-import org.kohsuke.args4j.CmdLineParser
-import org.kohsuke.args4j.Option
 
 import static org.apache.commons.lang.StringUtils.isNotBlank
-import static org.kohsuke.args4j.OptionHandlerFilter.ALL
 
+
+/**
+ * Core class that executes the StubRunner functionality
+ */
 @Slf4j
+@CompileStatic
 class StubRunner implements StubRunning {
 
-    private static final ThreadLocal<StubRunnerExecutor> stubRunner = new ThreadLocal<>()
-
-    @Option(name = "-r", aliases = ['--repository'], usage = "Path to repository containing the 'repository' folder with 'project' and 'mapping' subfolders (e.g. '/home/4finance/stubs/')", required = true)
-    private String repositoryPath
-
-    @Option(name = "-p", aliases = ['--projectPath'], usage = "Relative path to the project which you want to run (e.g. '/com/ofg/foo/barProject.json')", forbids = ['-a'])
-    private String projectRelativePath
-
-    @Option(name = "-a", aliases = ['--runAllStubs'], usage = "Switch that signifies that you want to run all stubs (e.g. 'true')", forbids = ['-p'])
-    private String runAllStubs
-
-    @Option(name = "-z", aliases = ['--zookeeperPort'], usage = "Port of the zookeeper instance (e.g. 2181)", forbids = ['lz'])
-    private Integer testingZookeeperPort = 2181
-
-    @Option(name = "-zl", aliases = ['--zookeeperLocation'], usage = "Location of local Zookeeper you want to connect to (e.g. localhost:23456)", forbids = ['-z'])
-    private String zookeeperLocation
-
-    @Option(name = "-minp", aliases = ['--minPort'], usage = "Minimal port value to be assigned to the Wiremock instance (e.g. 12345)", required = true)
-    private Integer minPortValue
-
-    @Option(name = "-maxp", aliases = ['--maxPort'], usage = "Maximum port value to be assigned to the Wiremock instance (e.g. 12345)", required = true)
-    private Integer maxPortValue
-
-    @Option(name = "-c", aliases = ['--context'], usage = "Context for which the project should be run (e.g. 'pl', 'lt')", required = true)
-    private String context
-
-    @Option(name = "-n", aliases = ['--serviceName'], usage = "Name of the service for which the project should be run (e.g. 'com/service/name')")
-    private String serviceName
+    private static final ThreadLocal<StubRunnerExecutor> STUB_RUNNER = new ThreadLocal<>()
```

Comment:
Or even a name suggesting that is it a container for a stub runner (not stubRunner itself).